model_analyzer/cli/cli.py

```diff
@@ -169,6 +182,7 @@ def _preprocess_and_verify_arguments(self, args):
                       " Specified Triton endpoints will be ignored.")
             args.triton_http_endpoint = 'localhost:8000'
             args.triton_grpc_endpoint = 'localhost:8001'
+            args.triton_metrics_url = 'http://localhost:8002/metrics'
```

Comment:
We don't need this anymore. We can go with defaults for now. I deleted this if branch in my end-to-end test PR.